source/templates/partials/_modal-test.twig

```diff
@@ -15,14 +15,30 @@
 
 <div class="form__wrapper">
   <label class="form__label" for=""></label>
-  <input class="form__field" id="" type="date" name="" value="">
+  <input class="form__field" id="" type="text" name="" value="" aria-describedby="calendar">
+  <p id="calendar"></p>
 </div>
 
 <div class="form__wrapper">
-  <select class="form__field" class="field__" id="" name="" size="">
-    <option class="form__field--oprion" value="">Option-1</option>
-    <option class="form__field--oprion" value="">Option-2</option>
+  <label class="form__label" for=""></label>
+  <select class="form__field" id="" name="">
+    <option value="">Option-1</option>
+    <option value="">Option-2</option>
   </select>
 </div>
 
+{# The error state #}
+<div class="form__wrapper">
+  <label class="form__label" for=""></label>
+  <input class="form__field form__fielmd--error" id="" type="text" name="" value="" aria-required="true" aria-describedby="error">
+  <p class="error"></p>
```

Comment:
- Be careful, `aria-describedby` needs to have the value if an ID to know which text is linked to this input, replace your `class`
- We can format ID for error message like that "error-<id-input>" (like that we are sure this error message is unique and we understand directly for which input it is for). For example `error-name` or `error-age`